README.md

```diff
@@ -256,4 +261,12 @@ g.add_vault_secret_path(path="your-secret-pat", mount_path="mount-point")
 
 Lastly, use the `fetch_vault_secret` function to fetch the secrets from your vault cluster.
 
-Raises a RuntimeError, if the path provided in `add_vault_secret_path` is invalid or the mount is invalid
+Raises a RuntimeError, if the path provided in `add_vault_secret_path` is invalid or the mount is invalid or anything else would be a generic Runtime Error
+
+## Dynamic Secrets
```

Comment:
Why can't we use other dynamic secrets engines? What is specific to databases?

Reply:
After discussion, yes we can use other dynamic secrets engines